pkg/sql/conn_executor.go

```diff
@@ -3852,6 +3843,34 @@ func (ex *connExecutor) initEvalCtx(ctx context.Context, evalCtx *extendedEvalCo
 	evalCtx.copyFromExecCfg(ex.server.cfg)
 }
 
+// maybeInitPCRReaderCatalog leases the system database to determine if
```

Comment:
nit: the comment should say `initPCRReaderCatalog`, not `maybeInitPCRReaderCatalog`